src/main/java/com/ericsson/eiffel/remrem/shared/MsgService.java

```diff
@@ -4,5 +4,12 @@
 import com.google.gson.JsonObject;
 
 public interface MsgService {
+    
+    String EVENT_ID = "eventId";
+    String ID = "id";
+    String META = "meta";
+    String EIFFEL_MESSAGE_VERSIONS = "eiffelMessageVersions";
+    
     String generateMsg(String msgType, JsonObject bodyJson);
+    String getEventId(JsonObject bodyJson);
```

Comment:
Since this is an interface we should document the methods according to java standards.